action.yml

```diff
@@ -7,6 +7,13 @@ inputs:
   sources:
     description: 'String with folders containing md files separated by commas (Supports glob pattern)'
     required: true
+  name:
+    description: 'Unique name for the specific output job'
+    required: true
+  top:
```

Comment:
What if there are multiple tops? 🥹

Reply:
Then they will go on top of each other :D